test/runnable/structlit.d

```diff
@@ -942,8 +942,8 @@ union X3991
 
 union Y3991
 {
-    int   a = void;
     dchar b = 'a';
+    int   a = void;
```

Comment:
?

Reply:
You cannot initialize a union member if it is not the first member of the union or if it's not void initialized (`a` void initializer is useless in this case).

Reply:
Oh, I thought that was a stack frame, my bad.